core/src/main/scala/org/apache/spark/h2o/H2OContext.scala

```diff
@@ -61,6 +61,7 @@ class H2OContext private (@transient val sparkContext: SparkContext, @transient
   with Serializable with SparkDataFrameConverter with SupportedRDDConverter with H2OContextUtils{
   self =>
 
+  val sqlc: SQLContext = SQLContext.getOrCreate(sparkContext)
```

Comment:
I am changing this val into `@transient` to make sure we are not capturing any non-serializable entities.